### PR TITLE
PHOENIX-6173 Archive test artifacts in Jenkins multibranch postcommit…

### PR DESCRIPTION
**Pull request by stoty**
… job

### CODE REVIEW

**Comment by stoty** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 32s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  shelldocs  |   0m  1s |  There were no new shelldocs issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 12s |  The patch does not generate ASF License warnings.  |
|  |   |   1m 22s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-911/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/911 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux b672ff115c47 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / 10ef717 |
| Max. process+thread count | 48 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-911/1/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.7.0 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: Jenkinsfile [hunk 1]
```diff
@@ -71,15 +71,19 @@ pipeline {
...
+                                    ulimit -a
+                                    mvn clean verify -Dhbase.profile=${HBASE_PROFILE} -B
+                                """
+                            }
                         }
                         post {
                             always {
-                                junit '**/target/surefire-reports/TEST-*.xml'
-                                junit '**/target/failsafe-reports/TEST-*.xml'
+                                archiveArtifacts artifacts: "HBASE_${HBASE_PROFILE}/**/target/surefire-reports/*.txt,**/target/failsafe-reports/*.txt,**/target/surefire-reports/*.dumpstream,**/target/failsafe-reports/*.dumpstream,**/target/surefire-reports/*.dump,**/target/failsafe-reports/*.dump"
```
Comment (richardantal) [inline Jenkinsfile:84]: I think HBASE_${HBASE_PROFILE}/ should be before the **/target everywhere.
Maybe you could split it to multiple lines.

Comment (stoty) [reply Jenkinsfile]: Thanks, done that.

**Comment by stoty** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  shellcheck  |   0m  0s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  shelldocs  |   0m  1s |  There were no new shelldocs issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 12s |  The patch does not generate ASF License warnings.  |
|  |   |   1m 21s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-911/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/phoenix/pull/911 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux f6f160bd7741 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev/phoenix-personality.sh |
| git revision | master / 10ef717 |
| Max. process+thread count | 47 (vs. ulimit of 30000) |
| modules | C: . U: . |
| Console output | https://ci-hadoop.apache.org/job/Phoenix/job/Phoenix-PreCommit-GitHub-PR/job/PR-911/2/console |
| versions | git=2.7.4 maven=3.3.9 shellcheck=0.7.0 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by richardantal** [comment]
+1

**Pull request closed without merging**